version/version.go

```diff
@@ -4,5 +4,5 @@
 
 package version
 
-// Version describes the version of build
-var Version = "0.0.1"
+// Version describes the version of Shipwright build controller
+var Version = "0.3.0"
```

Comment:
Is this something we need to make sure to update at every release? In Tekton I think we have a similar const that's set to "devel", which we set with ldflags when we release.

Reply:
Good question.

I also think it is not good to update every time, but the `0.0.1` version is not correct for us.

I saw Tekton uses `devel` here: https://github.ibm.com/coligo/tekton-pipeline/blob/9f40236757d94cccacbb10d48bca48461a39e229/cmd/controller/main.go#L54

I think it should be better for us too. 

Let us see how other guys think.

If no any other `version` suggestion. I will change to `devel` soon.

Thanks!

Reply:
+1 on the tekton approach

in addition to the link @zhangtbj mentioned the version set in the ldflags comes from git related metadata 

let us know @zhangtbj if you need pointers for that as well, assuming you want to take that on here, or if you just want to do the "devel" piece in main.go for now, that is OK too

Reply:
Already changed to `devel`

Hi @gabemontero ,

As I know the version which I pointed is also used for the `version` file: https://github.com/tektoncd/pipeline/blob/master/pkg/version/version.go#L19

And I just would like to correct the version in the controller log, like this:
```
✔ $ operator-sdk run --local
INFO[0000] Running the operator locally in namespace default. 
{"level":"info","ts":1614657222.882747,"logger":"build","msg":"Shipwright Build Controller Version: devel"}
{"level":"info","ts":1614657222.8828,"logger":"build","msg":"Go Version: go1.13.15"}
{"level":"info","ts":1614657222.882808,"logger":"build","msg":"Go OS/Arch: darwin/amd64"}
{"level":"info","ts":1614657222.882815,"logger":"build","msg":"Version of operator-sdk: v0.18.2"}
```

So just modify in the `version.go` is enough for me.

Thanks all!

Reply:
sounds good